typescript/phoenix-sdk/src/market.ts

```diff
@@ -245,4 +246,23 @@ export class Market {
       inAmount,
     });
   }
+
+  getPriceDeicmalPlaces(): number {
```

Comment:
typo in name

where is this used?

Reply:
good call, Logan recommended that this was a better place for it (for used in data ui)